### ISSUE TITLE
[SCREENREADER]: GIBCT EYB - Focus MUST be set on the first added item when Show Next 10 button is pressed

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9953]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The non-VET TEC school locations table allows users to show more results using Show next 10 and Show all buttons. This feature is also available on the VET TEC Additional Programs list.

Focus remains on the button when users click or keypress. This could cause a a situation where screen reader users do not know information has been added to the page, where it was added before the button instead of after. Screen shot attached below.

## Related Issues
* #9952 

## Point of Contact

**VFS Point of Contact:** _Trevor, Crystabel_

## Acceptance Criteria

- [ ] Focus is set on the first link or text of the first added row
- [ ] Screen readers announce the added item properly. This will probably be only the first cell of information, but should give screen reader users an understanding things have changed.
- [ ] Tables should announce the correct number of rows after appending the new rows
- [ ] No axe errors when the axe plugin is run or CI/CD axe checks are run

## Environment

* /gi-bill-comparison-tool/profile/21852735 (this is a sample profile view with the right information)

## WCAG or Vendor Guidance (optional)
* [Meaningful Sequence: Understanding SC 1.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html)

## Screenshots or Trace Logs

![Screen Shot 2020-06-08 at 11 50 32 AM](https://user-images.githubusercontent.com/934879/84060509-fb499400-a981-11ea-8aa5-51ef6e5e6db9.png)

**Comment by zurbergram** [comment]
The "Approved Programs" accordion will also have this issue, see https://github.com/department-of-veterans-affairs/va.gov-team/issues/9722

**Comment by hughes-dustin** [comment]
branch: bah-9953
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/13045

**Comment by sonntag-adam** [comment]
@1Copenut this change is now in staging.

**Comment by desireeturner** [comment]
@Clark-W-Griswold This issue is ready for your review on staging. Thank you!

**Comment by 1Copenut** [comment]
@desireeturner We will need to take another look at this item. It's not setting focus on the right element when I view it on a mobile device or at mobile width. I think it has to do with replacing the `<table>` with an unordered list. I'm attaching a screen shot below and will open an improvement ticket to use our new responsive table when it's ready. Work on that item is currently in progress.

---
![Screen Shot 2020-07-08 at 4 25 32 PM](https://user-images.githubusercontent.com/934879/86972080-f2421300-c137-11ea-8a07-91184f43ebcc.png)

**Comment by 1Copenut** [comment]
@desireeturner Here is the work for the new responsive table component: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203

**Comment by desireeturner** [comment]
@hughes-dustin moving back to in progress so you can review Trevor's feedback.

**Comment by desireeturner** [comment]
Moved to Icebox until the new responsive table component is available.

**Comment by desireeturner** [comment]
Moved to blocked until the following issue is complete: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203

**Comment by 1Copenut** [comment]
@desireeturner I wanted to follow up quickly and make sure you saw my comment https://github.com/department-of-veterans-affairs/va.gov-team/issues/11055#issuecomment-675042916 on #11055 last week. We have a working prototype of the responsive table that another VFS team is working on. The official component won't be ready in the design system for awhile, but the code is working, with a few changes I outlined.

That code could be used as a baseline for your responsive table and the refactor requested in this ticket, instead of it being blocked waiting for an official design system component.

**Comment by desireeturner** [comment]
@hughes-dustin reviewed and confirmed that we can go ahead with a solution based on @1Copenut 's comment above.

**Comment by delli-gatti-michael** [comment]
@1Copenut This appears to have been fixed by the fix for #11055 which is on staging now.

**Comment by 1Copenut** [comment]
Retested in staging with preferred screen reader pairings. This is behaving as expected, moving to close.

/cc @desireeturner
